wdl-format/src/token.rs

```diff
@@ -52,6 +52,31 @@ impl<T: Token> TokenStream<T> {
             let _ = self.0.pop();
         }
     }
+
+    /// Returns whether the stream is empty.
+    pub fn is_empty(&self) -> bool {
+        self.0.is_empty()
+    }
+
+    /// Returns an iterator over the tokens in the stream.
+    pub fn iter(&self) -> std::slice::Iter<'_, T> {
+        self.0.iter()
+    }
+
+    /// Clears the stream.
+    pub fn clear(&mut self) {
+        self.0.clear();
+    }
+
+    /// Drains the stream.
+    pub fn drain(&mut self) -> std::vec::Drain<'_, T> {
+        self.0.drain(..)
+    }
```

Comment:
Am I correct that this is `clear`, but it returns the items from the stream?

Reply:
Exactly. I was using this in a WIP implementation, but don't use it now. We can remove if we want.